docs/source/reference/slurm/slurm-getting-started.rst

```diff
@@ -0,0 +1,240 @@
+.. _slurm-getting-started:
+
+Getting Started on Slurm
+========================
+
+.. note::
+
+    Slurm support is currently in **early access**. If you're interested in trying it out,
+    please `fill out this form <https://forms.gle/rfdWQcd9oQgp41Hm8>`_ to get access.
+
+Quickstart
+----------
+Have SSH access to a Slurm cluster? Get started with SkyPilot in 3 steps:
+
+.. code-block:: bash
+
+   # 1. Configure your Slurm cluster in ~/.slurm/config
+   $ mkdir -p ~/.slurm && cat > ~/.slurm/config << EOF
+   Host mycluster
+       HostName login.mycluster1.myorg.com
+       User myusername
+       IdentityFile ~/.ssh/id_rsa
+   EOF
+
+   # 2. Verify SkyPilot detects your Slurm cluster
+   $ sky check
+   # Shows "Slurm: enabled"
+
+   # 3. Launch your first SkyPilot task
+   $ sky launch --gpus H100:1 -- nvidia-smi
+
+For detailed instructions, prerequisites, and advanced features, read on.
+
+Prerequisites
+-------------
+
+To connect and use a Slurm cluster, SkyPilot needs SSH access to the Slurm login node (where you can run ``sbatch``, ``squeue``, etc.).
+
+In a typical workflow:
+
+1. A cluster administrator sets up a Slurm cluster and provides users with SSH access to the login node.
+
+2. Users configure their ``~/.slurm/config`` file with connection details for their Slurm cluster(s).
+   SkyPilot reads this configuration file to communicate with the cluster.
```

Comment:
Q: Do we need to mention that `~/.slurm/config` is a SkyPilot specific concept? Since I believe most people would probably just configure the connection details in `~/.ssh/config`.

Reply:
I was thinking it's covered below in `Configuring Slurm clusters`, so might be ok to not go into detail here. Updated the wording a bit, wdyt?